src/components/Calendar/Filters.tsx

```diff
@@ -47,7 +45,7 @@ const CalendarFilters: React.FC<CalendarFilterAccordionProps> = ({
     ): void => {
         const newLocations = event.target.checked
             ? currentLocations.concat(location)
-            : currentLocations.filter((currentLocation) => currentLocation !== location);
+            : currentLocations.filter((testLocation) => testLocation !== location);
```

Comment:
Not fussed at all, but I don't think we needed to change this name
```suggestion
            : currentLocations.filter((currentLocation) => currentLocation !== location);
```